libbuteosyncfw/profile/SyncLog.cpp

```diff
@@ -121,7 +121,7 @@ SyncLog::SyncLog(const QDomElement &aRoot)
     }
 
     // Sort result entries by sync time.
-    //qSort(d_ptr->iResults.begin(), d_ptr->iResults.end(), syncResultPointerLessThan);
+    //std::sort(d_ptr->iResults.begin(), d_ptr->iResults.end(), syncResultPointerLessThan);
```

Comment:
Why do we keep these commented lines?

Reply:
No idea, but wanted to keep the question whether to keep them or not separate. Just migrated the content now as mechanic change so nobody gets bad ideas later.